Src/BootCamp.Chapter/Lesson4.cs

```diff
@@ -0,0 +1,178 @@
+using System;
+
+namespace BootCamp.Chapter
+{
+    public static class Lesson4
+    {
+        public static void Demo()
+        {
+            WelcomePrompt();
+            do
+            {
+                PromptUserForStats();
+
+            } while (PromptCalculateNewBMI());
+        }
+        public static bool PromptCalculateNewBMI()
+        {
+            Console.WriteLine("Do you wish to calculate another person? Y/N ");
+            string response = Console.ReadLine();
+            return response.StartsWith("Y") || response.StartsWith("y");
+        }
+
+        public static void PromptUserForStats()
+        {
+
+            string name = PromptNameInput("What is your name?");
+            int age = PromptAgeInput("How old are you? If I may ask");
+            float weight = PromptFloatInput("What is your weight? In Kilograms please.");
+            float height = PromptFloatInput("How tall are you? In Meters please.");
+
+            bool isHeightAndWeightValid = ValidateHeightAndWeight(weight, height);
+
+            if (isHeightAndWeightValid)
+            {
+                float calculatedBMI = GetBmi(weight, height);
+                Console.WriteLine($"{name} is {age} years old, weight is {weight} kg with a height of {height} meters.");
+                Console.WriteLine($"The calculated BMI  is: {calculatedBMI:F}");
+            }
+            else
+            {
+                Console.WriteLine("The values you provided were not valid.");
+            }
+
+        }
+
+        private static void WelcomePrompt()
+        {
+            Console.WriteLine("+------------------------------+");
+            Console.WriteLine("|                              |");
+            Console.WriteLine("|    Welcome to C# Bootcamp    |");
+            Console.WriteLine("|        BMI Calculator        |");
+            Console.WriteLine("|        Version 0.0.3         |");
+            Console.WriteLine("|                              |");
+            Console.WriteLine("+------------------------------+");
+        }
+
+        public static string GetInput()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Consider making method 'GetInput' a property.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4884658)